Update version_notes.adoc with link to new mailing list (#934)
starknet-io/starknet-docs: components/Starknet/modules/starknet_versions/pages/version_notes.adoc

[id="upcoming"]
= Starknet release notes

> The following release notes cover the ongoing version changes to Starknet. To get the latest version updates delivered to your inbox, please subscribe to our link:https://www.starknet.io/en/roadmap[mailing list].

Reviewer: ```suggestion
The following release notes cover the ongoing version changes to Starknet. You can subscribe to get the latest version updates delivered to your inbox at  link:https://www.starknet.io/en/roadmap[Starknet Roadmap & version updates].
```
